### PR TITLE
replace nose tools with assert - just /olympia/addons/

### PR DESCRIPTION
**Pull request by eviljeff**

### CODE REVIEW

**Comment by eviljeff** [comment]
to save everyone's sanity, this will be an app-by-app process

**Comment by EnTeQuAk** [comment]
I already started doing that some time ago and going app by app is definetly a very good choice. Also, there's https://github.com/pytest-dev/unittest2pytest that might help with some automatic conversion.

File: src/olympia/addons/tests/test_views.py [hunk 1]
```diff
@@ -1076,18 +1075,18 @@ def test_other_personas(self):
 
         other = addon_factory(type=amo.ADDON_PERSONA)
         self.create_addon_user(other)
-        eq_(other.status, amo.STATUS_PUBLIC)
-        eq_(other.disabled_by_user, False)
+        assert other.status == amo.STATUS_PUBLIC
+        assert not other.disabled_by_user
 
         # TODO(cvan): Uncomment this once Personas detail page is impalacized.
```
Comment (diox) [inline src/olympia/addons/tests/test_views.py]: lol.

**Comment by diox** [comment]
r+

File: src/olympia/addons/tests/test_views.py (hunk 1 above)
Comment (eviljeff) [inline src/olympia/addons/tests/test_views.py]: we should have a TODO bounty program

**Pull request merged**